### ISSUE TITLE
Step List with >9 steps (formatting issue)

### ISSUE THREAD

**Issue by openbuilds-engineer** [olton/Metro-UI-CSS#1328]
Hi 

In the step list component: 

![step list](https://user-images.githubusercontent.com/31449271/52954374-b7004200-3392-11e9-8cd8-5a2a732329c8.PNG)

If you have more than 9 steps (ie 10, 11, etc with two digits) theres a problem with formatting and the number overlaps

**Comment by openbuilds-engineer** [comment]
My temporary workaround:

```
.step-list > li {
  margin-left: 40px;
}

.step-list > li::before {
  margin-left: -40px;
}
```

**Comment by olton** [comment]
Thank you

**Comment by olton** [comment]
I fix it in 4.2.37 for three digits in the index, but this decision does not quite suit me.
